src/service/video-manager-impl.js

```diff
@@ -437,6 +556,216 @@ class VideoEntry {
   }
 
   /**
+   * Called when the video's position in the viewport changed (at most once per
+   * animation frame)
+   * @param {PositionInViewportEntryDef} newPos
+   * @private
+   */
+  onDockableVideoPositionChanged_(newPos) {
+
+
+    const inViewportHeight = this.calculateDockableVideoPosition_(newPos);
```

Comment:
I believe that should stay there because even if the video is not docked yet, we should update its `minimizePosition` from default to inview/top/bottom.